database/migrations/2024_01_22_233137_prepare_tables_for_next _year.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up()
+    {
+        // Drop the 'reservations' table if it exists before creating a new one
+        Schema::dropIfExists('reservations');
+        
+        // Create a copy of the 'r2023_rezervace' table and its data
+        DB::statement('CREATE TABLE reservations LIKE r2023_rezervace');
+        DB::statement('INSERT INTO reservations SELECT * FROM r2023_rezervace');
```

Comment:
You want only table schema. If you copy data, reservations will be full, and this data will be sent to FE, there is calc for dashboard page counting money and so on